src/main/java/frc/robot/commands/AutoBalance.java

```diff
@@ -4,35 +4,43 @@
 
 package frc.robot.commands;
 
-import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.PIDCommand;
 import frc.robot.subsystems.DriveTrainSubsystem;
 
-public class AutoBalance extends CommandBase {
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
+public class AutoBalance extends PIDCommand {
   /** Creates a new AutoBalance. */
-  DriveTrainSubsystem m_driveTrain;
+  private DriveTrainSubsystem m_driveTrain; 
+
 
   public AutoBalance(DriveTrainSubsystem driveTrain) {
-    m_driveTrain = driveTrain;
+    super(
+        // The controller that the command will use
+        new PIDController(0, 0, 0), // Need to insert proper pid values here, waiting until testing
+        // This should return the measurement
+        () -> driveTrain.robotPitch(),
+        // This should return the setpoint (can also be a constant)
+        0,
+        // This uses the output
+        output -> {
+          // Use the output here
```

Comment:
you said the same thing twice in the comments

Reply:
he needs to make sure you know.